### ISSUE TITLE
Reoccurring error on opening connection to Azure Machine Learning Compute Instance

### ISSUE THREAD

**Issue by evakkuri** [microsoft/vscode-tools-for-ai#1541]
Does this occur consistently?  Yes
Repro steps:

1. Open remote connection to Azure Machine Learning Compute Instance

This does not seem to cause any issues, but it's annoying to see the error message every time.

Action: azureAccount.onSubscriptionsChanged
Error type: REQUEST_SEND_ERROR
Error Message: request to redacted:url failed, reason: getaddrinfo ENOTFOUND redacted:idworkspace.westeurope.api.azureml.ms

Version: 0.8.2
OS: linux
OS Release: 5.4.0-1068-azure
Product: Visual Studio Code
Product Version: 1.66.1
Language: en

<details>
<summary>Call Stack</summary>

```
new t extension.js:2:486489
t.<anonymous> extension.js:2:470040
extension.js:2:2450576
Object.throw extension.js:2:2450681
c extension.js:2:2449471
```

</details>

**Comment by sevillal** [comment]
@evakkuri thanks for filing this issue. Is this happening every time you open a remote connection? Are you connecting to Compute Instance through the ML Studio?

**Comment by evakkuri** [comment]
Currently this happens every time I connect. I'm not connecting via ML Studio, instead through VS Code with the Azure Machine Learning extension.

**Comment by sevillal** [comment]
@evakkuri we have published version v0.10.0 of the extension. Could you please upgrade and retry to check if you issue is still reproducible? Please reopen this issue if that's the case.
